### PR TITLE
TIP-3804 Add databaseResultsRestricted to project new/update

### PR DESCRIPTION
**Pull request by durae**
Depends on https://github.com/dnanexus/nucleus/pull/11135

### CODE REVIEW

**Review by austinlocke — APPROVED** [review]
LGTM

**Review by ksoni — APPROVED** [review]
Looks good to me.

**Comment by durae** [comment]
Hi @kpjensen / @wormsik ! Can I get a quick look from any of you before merge? Thanks!

File: src/python/dxpy/bindings/dxproject.py [hunk 1]
```diff
@@ -380,6 +387,8 @@ def update(self, name=None, summary=None, description=None, protected=None,
         :type database_ui_view_only: boolean
         :param external_upload_restricted: If provided, whether project members can upload data to project from external sources, e.g. outside of job
         :type external_upload_restricted: boolean
+        :param database_results_restricted: If provided, minimum amount of data that project members with VIEW access can see from databases in the project
```
Comment (wormsik) [inline src/python/dxpy/bindings/dxproject.py:390]: seems that `unset_database_results_restricted` documentation is missing

**Review by kpjensen — APPROVED** [review]

**Review by wormsik — CHANGES_REQUESTED** [review]

File: src/python/dxpy/bindings/dxproject.py (hunk 1 above)
Comment (durae) [reply src/python/dxpy/bindings/dxproject.py:392]: Fixed it for unset_database_results_restricted and unset_allowed_executables. Thanks!

**Review by wormsik — APPROVED** [review]
LGTM 🚀

**Pull request merged**